### PR TITLE
Made some more compatibility fixes to support the latest raw ops.

### PR DESCRIPTION
**Pull request by eaplatanios**
Hopefully fixes all remaining issues [here](https://github.com/apple/swift/pull/23700).

### CODE REVIEW

File: Sources/DeepLearning/Operators.swift [hunk 1]
```diff
@@ -133,6 +133,35 @@ public extension Padding {
     }
 }
 
+/// An older padding scheme. Used by padding, convolution, and pooling ops.
+// @_frozen // SR-9739
+public enum PaddingV1 {
```
Comment (rxwei) [inline Sources/DeepLearning/Operators.swift]: We should not introduce versioned types in the deep learning library. Why is this necessary? All ops should use the general `Padding` type.

Also, I don't think the extension below compiles. Could you verify things compile and work locally?

Comment (eaplatanios) [reply Sources/DeepLearning/Operators.swift]: The pooling ops do not support explicit paddings. We could either not support explicit paddings at all for now (i.e., not even for convolution ops), or have two different padding modes.

I checked and it compiled locally but I may have mixed up my branches. I'll check again.

Comment (rxwei) [reply Sources/DeepLearning/Operators.swift]: In that case, let’s fall back to not supporting explicit paddings for now.

Comment (eaplatanios) [reply Sources/DeepLearning/Operators.swift]: I just pushed changes removing the explicit padding mode.

**Comment by Shashi456** [comment]
@eaplatanios if this is set to get merged earlier than #70, could you make the operators directory in this one. I could then get started on #54 to avoid a conflict.

**Comment by eaplatanios** [comment]
This needs a couple more fixes I'll do in a bit.

**Comment by eaplatanios** [comment]
@rxwei this now compiles fine on my machine. Sorry for the delays but I've been super busy with preparing my thesis proposal.
